dotCMS/src/integration-test/java/com/dotmarketing/startup/runalways/Task00050LoadAppsSecretsTest.java

```diff
@@ -21,6 +21,7 @@
 import com.dotmarketing.exception.DotDataException;
 import com.dotmarketing.exception.DotSecurityException;
 import com.dotmarketing.portlets.contentlet.business.HostAPI;
+import com.dotmarketing.startup.runalways.Task00050LoadAppsSecrets;
```

Comment:
Codacy found an issue: [No need to import a type that lives in the same package](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6392590)